include/miopen/miopen.h

```diff
@@ -5324,6 +5324,7 @@ typedef enum
     miopenTensorSoftmaxDY    = 14,
 #endif
 
+    miopenScalarArgument = 1 << 16,
```

Comment:
```suggestion
    miopenTensorArgumentIsScalar = 1U << 31,
```
### Why renamed
Use common prefix on a new enum member. Looks terrible and irrational, unless we rename enum
- `miopenTensorArgumentId_t` -> `miopenArgumentId_t`

and then members
- `miopenTensorConvolutionX` -> `miopenArgumentTensorConvolutionX`

etc, but this is what we have right now.

### Why 16 to 31

- Pros: Larger contiguous range of spare ID values
- Cons: None (no cost at runtime)

Why not 32 but 31 and why 1U you already know I guess ;)

Reply:
AFAIK we can't rename enum, sadly.
I don't know why I even chose 16, great catch.